packages/d3fc-webgl/src/shaders/fragmentShaderSnippets.js

```diff
@@ -82,15 +94,27 @@ export const ohlc = {
 export const area = {
     header: `
         varying float vDefined;`,
-    body: `if (vDefined < 0.5) {
+    body: `
+        float canFill = 1.0;
+        float canStroke = 0.0;
+        if (vDefined < 0.5) {
             discard;
         }
         gl_FragColor = vec4(0.86, 0.86, 0.86, 1);`
 };
 
 export const boxPlot = {
-    header: `varying float vDefined;`,
+    header: `
+        varying float vDefined;
+        varying float vCanFill;
+    `,
     body: `
+        float canFill = vCanFill;
+        float canStroke = 1.0 - vCanFill;
+
+        vec4 defaultFillColor = vec4(0.86, 0.86, 0.86, 1.0);
+        gl_FragColor = (canFill * defaultFillColor) + ((1.0 - canFill) * gl_FragColor);
```

Comment:
`(1.0 - canFill)` can be replaced with `canStroke`

Reply:
Done.